Formula/podman.rb

```diff
@@ -62,21 +63,22 @@ class Podman < Formula
 
   resource "netavark" do
     on_linux do
-      url "https://github.com/containers/netavark/archive/refs/tags/v1.3.0.tar.gz"
-      sha256 "cc8a8e03498cb9b4c74fdbda09a64fdf9000fea398d07073c4e368fc83d35f56"
+      url "https://github.com/containers/netavark/archive/refs/tags/v1.5.0.tar.gz"
+      sha256 "303fbcf3fc645b0e8e8fc1759626c92082f85f49b9d07672918aebd496a24d34"
     end
   end
 
   resource "aardvark-dns" do
     on_linux do
-      url "https://github.com/containers/aardvark-dns/archive/refs/tags/v1.3.0.tar.gz"
-      sha256 "6dd1ce4346ed5c57bbd990140e02e69c036919032582b937d2ad7835329d3bc3"
+      url "https://github.com/containers/aardvark-dns/archive/refs/tags/v1.5.0.tar.gz"
+      sha256 "b7e7ca1b94c1a62c8800f49befb803ec37cc5caf7656352537343a8fb654e4a6"
     end
   end
 
   def install
     if OS.mac?
       ENV["CGO_ENABLED"] = "1"
+      ENV.prepend_path "PATH", Formula["make"].opt_libexec/"gnubin"
```

Comment:
Why is `make` defined as a dependency on Linux above and it is prepended to PATH only on macOS? Also it should not be necessary to install brewed make on Linux, we require it to be installed system-wide, macOS on the other hand - sure.

Reply:
I was meant to add `make` to the `on_macos` block but added it to `on_linux` by mistake. Thanks!